### DISCUSSION TITLE
Attach mp4 and other formats in Assistance API

### DISCUSSION THREAD

**Discussion by YourKey** [danny-avila/LibreChat#2828] [Feature Requests & Suggestions]
In GPTs I can upload different files and the chat processes them. In Librechat, the Assistance API does not allow me to download an mp4 file. Tell me, will it be possible to process different formats?
I'm using the latest Librechat update with Assistance api v2 and gpt-4o.

![2024-05-22_19-10-17](https://github.com/danny-avila/LibreChat/assets/10841762/40caaa07-c316-4346-a423-1a0b4a6bcaa4)

**Comment by danny-avila** [comment]
https://platform.openai.com/docs/assistants/tools/code-interpreter/supported-files
